src/gui/qgsmapmouseevent.h

```diff
@@ -94,6 +94,12 @@ class GUI_EXPORT QgsMapMouseEvent : public QMouseEvent
     inline QgsPoint mapPoint() const { return mMapPoint; }
 
     /**
+      * Returns the matching data from the most recently snapped point.
+      * @return the snapping data structure
+      */
+    QgsPointLocator::Match mapPointMatch() const { return mSnapMatch; }
```

Comment:
Please also add a note that the method was added in QGIS 2.14 so plugin developers know the requirements if they want to use the function + add the method to .sip file to make it available in Python.